index.js

```diff
@@ -64,12 +64,18 @@ async function loadGlobalMixin(helpers, globs) {
 }
 
 function addGlobalMixins(helpers, local, global, parent) {
-  for (let name in global) {
+  let dirsOfMixins = Object.values(global).map(({ file }) => dirname(file))
+  let uniqueDirsOfMixins = Array.from(new Set(dirsOfMixins))
+
+  uniqueDirsOfMixins.forEach(dirOfMixin => {
     helpers.result.messages.push({
-      type: 'dependency',
-      file: global[name].file,
+      type: 'dir-dependency',
+      dir: dirOfMixin,
       parent: parent || ''
```

Comment:
1. `dir-dependnecy` should not replace `dependency`. `dependency` is for loaded files. `dir-dependency` is for new files.
2. `dir-depedency` need `glob` key